icons/templates/icon-template.js

```diff
@@ -12,15 +12,16 @@ function template(
     const tpl = template.smart({ plugins })
 
     return tpl.ast`
-        ${imports}
         import PropTypes from 'prop-types'
+        ${imports}
```

Comment:
This is an unrelated change to appease the linter. `React` was imported before `PropTypes` which caused a import order ESLint error. Since these are generated files, probably nobody cares, but at least I will sleep better at night knowing that our generated files adhere to our ESLint rules.